src/status_im/ui2/screens/chat/home.cljs

```diff
@@ -37,7 +36,8 @@
             [status-im.ui2.screens.chat.components.message-home-item.view :refer [messages-home-item]]
             [status-im.ui2.screens.chat.components.contact-item.view :refer [contact-item]]
             [clojure.string :as str]
-            [status-im2.common.plus-button.view :as components.plus-button])
+            [status-im2.common.plus-button.view :as components.plus-button]
+            [status-im2.setup.config :as config])
```

Comment:
lint should fail here

Reply:
But lint didn't throw any errors.

![Screenshot 2022-11-17 at 9 55 37 PM](https://user-images.githubusercontent.com/19339952/202501633-cdb0ea93-dee2-4e26-8bf7-65cdce7ea334.png)

Reply:
oh i confused the file, sorry